MapView/Map/RMTileCache.m

```diff
@@ -423,6 +428,16 @@ - (void)beginBackgroundCacheForTileSource:(id <RMTileSource>)tileSource southWes
     }
 }
 
+- (id<RMTileCache>)databaseCacheAtPath:(NSString *)cachePath
+{
+    RMDatabaseCache *dbCache = [[RMDatabaseCache alloc] initWithDatabase:cachePath];
+    [dbCache setCapacity:LONG_MAX];
```

Comment:
Presumably we have in mind a strategy for managing the tiles? As this setup will essentially eat disk space until it runs out without ever purging.